pkgs/applications/emulators/libretro/cores/atari800.nix

```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  fetchFromGitHub,
+  mkLibretroCore,
+}:
+mkLibretroCore {
+  core = "atari800";
+  version = "0-0-unstable-2024-10-31";
```

Comment:
Ops!

Reply:
Done.

Run a `rg 0-0` to make sure that we don't have any more matches.